hutch_python/happi.py

```diff
@@ -118,6 +135,28 @@ def get_happi_objs(
         if device.name in exclude_devices:
             containers.remove(device)
 
+    # Load additional devices
```

Comment:
Maybe exclude should happen after additional so we can exclude from additional
This gets a little confusing but I think we should change the order